app/components/Notification/index.jsx

```diff
@@ -43,7 +52,15 @@ let NotifiItem = React.createClass({
         };
         let timeStyle = {
             fontSize: "0.9em"
-        }
+        };
+        let achorReadStyle = {
+            color: "#aaa",
+            cursor: "pointer"
```

Comment:
We need to follow material design style guideline:
http://www.google.co.jp/design/spec/style/color.html#color-ui-color-application

And we want to enable theme support so we should use theme variable:
`
this.context.muiTheme.palette.[colorName]
`